src/main/java/Parser.java

```diff
@@ -0,0 +1,171 @@
+        TaskList todoT = new Todo(num, "[✗]", word, "todo");
+        todoT.addList(todoT, array, num);
+    }
+
+    /**
+     * <p>
+     *    callEvent is used to handle event tasks
+     * </p>
+     * @param word get command
+     * @param num task number
+     * @param array the list of task
+     */
+
+    public void callEvent(String word, int num, 
+                          ArrayList<TaskList> array) {
+        String arr2[] = word.split("/at");
+        DateTime dT1 = new DateTime(arr2[1]);
+        TaskList eventT = new Event(num, "[✗]", arr2[0], "event", dT1);
+        eventT.addList(eventT, array, num);
+    }
+
+    /**
+     * <p>
+     *     callDeadline is used to handle deadline tasks
+     * </p>
+     * @param word get command
+     * @param num task number
+     * @param array the list of task
+     */
+    public void callDeadline(String word, int num, 
+                             ArrayList<TaskList> array) {
+        String arr3[] = word.split("/by");
+        String list[] = arr3[1].split(" ");
+        DateTime dT2 = new DateTime(arr3[1]);
+        TaskList deadlineT = new Deadline(num, "[✗]", 
+                                          arr3[0], "deadline", dT2);
+        deadlineT.addList(deadlineT, array, num);
+    }
+
+    /**
+     * <p>
+     *     callDelete is used to handle delete tasks request
+     * </p>
+     * @param word get command
+     * @param array the list of task
+     */
+
+    public void callDelete(String word, ArrayList<TaskList> array) {
+        String arr4[] = word.split(" ");
+        int taskNum = Integer.parseInt(arr4[1]);
+        TaskList forDelete = array.get(taskNum - 1);
+        forDelete.deleteTask(forDelete, array);
+      
+        int size = array.size();
+
+        for(int i = 0; i < size; i++) {
+            TaskList t = array.get(i);
+            int current = t.getTaskNumber();
+            String type = t.getType();
+
+            if(current != i + 1) {
+                if(type.equals("todo")) {
+                    TaskList t1 = new Todo(i + 1, t.getTaskCheck(), 
+                                           t.getTaskName(), "todo");
+                    array.set(i, t1);
+                } else if(type.equals("event")) {
+                    TaskList t2 = new Event(i + 1, t.getTaskCheck(), 
+                                            t.getTaskName(), "event", t.getAB());
+                    array.set(i, t2);
+                } else {
+                    TaskList t3 = new Deadline(i + 1, t.getTaskCheck(), 
+                                               t.getTaskName(), "deadline", t.getAB());
+
+                    array.set(i, t3);
+                }
+            }
+        }
+    }
+
+    public void callFind(String word, ArrayList<TaskList> array) {
+        ArrayList<TaskList> arr = new ArrayList<TaskList>();
+        String arr5[] = word.split(" ");
+        String searchWord = arr5[1];
+        int size = array.size();
+
+        for(int i = 0; i < size; i++) {
```

Comment:
why do you have to take it out and create a new one exactly like it? Just put the one you took out back in!